kong/db/declarative/init.lua

```diff
@@ -306,29 +306,32 @@ end
 local function post_upstream_crud_delete_events()
   local upstreams = kong.cache:get("upstreams|list", nil, nil_fn)
   if upstreams then
-    for _, id in ipairs(upstreams) do
-      local ok = kong.worker_events.post("crud", "upstreams", {
-        operation = "delete",
-        entity = {
-          id = id,
-          name = "?", -- only used for error messages
-        },
-      })
-      if not ok then
-        kong.log.err("failed posting invalidation event for upstream ", id)
-      end
+    local ok = kong.worker_events.post("crud", "upstreams", {
+      operation = "delete",
+      entity = "all",
```

Comment:
Wasn't it possible to latch on to the `flip_config` event and get rid of these additional `crud` events here altogether?

Reply:
Yep, it makes sense. Will do that.